ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -2888,17 +2888,74 @@ FEDataManager::updateQuadPointCountData(const int coarsest_ln, const int finest_
 std::vector<std::pair<Point, Point> >*
 FEDataManager::computeActiveElementBoundingBoxes()
 {
+    // Get the necessary FE data.
```

Comment:
Why was this method reverted to the old implementation?

Reply:
The only difference between the file here and the `master` version is in `computeActiveElementBoundingBoxes` function...I didn't dig too deep into this but clearly if I use the version on the master branch I get wrong results. So I think the new changes made in `FEDataManager.cpp ` on the `master` branch might have to be reviewed to make sure it works for surface mesh as well.

Reply:
This might be fixed by some bugs I addressed in #829. If not could you please put this in a separate PR?

Reply:
I just tried it with the newest version on master but the problem persists. It's only an issue when running in parallel btw...yup, I can create a separate PR for it.

Reply:
Thanks.

I think we can fix the problem with either #829 or another fix. I'm surprised that ex5 works at all in parallel given this issue. I'll spend some time today trying to get to the bottom of this.